runtime/ignite/iginite.go

```diff
@@ -119,6 +119,9 @@ func (c *IgniteRuntime) WithConfig(cfg *runtime.RuntimeConfig) {
 	}
 
 }
+func (c *IgniteRuntime) WithKeepMgmtNet() {
+	c.ctrRuntime.WithKeepMgmtNet()
```

Comment:
Can someone check if this makes sense? 
Do we need to do something else here?

Reply:
ignite runtime uses the docker runtime for management network, so that seems ok
@networkop is that right?

Reply:
yes, it makes sense